shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -87,10 +87,20 @@ public void createBridgeExecutor(ProcessorDTO processorDTO) {
 
             // create or update the secrets for the bridgeExecutor
             createOrUpdateBridgeExecutorSecret(bridgeExecutor, processorDTO);
-        } else if (existing.getStatus().isReady()) {
-            LOGGER.info("BridgeExecutor '{}' already exists and is ready. Notifying manager that it is ready.", processorDTO.getId());
-            ProcessorManagedResourceStatusUpdateDTO updateDTO =
-                    new ProcessorManagedResourceStatusUpdateDTO(processorDTO.getId(), processorDTO.getCustomerId(), processorDTO.getBridgeId(), ManagedResourceStatus.READY);
+        } else {
+            ManagedResourceStatus inferredStatus = existing.getStatus().inferManagedResourceStatus();
+            // The Controller would have notified the Manager with PROVISIONING before it first started.
```

Comment:
@manstis I'm a little unsure if this change is related to the `status_message` feature? Could you please clarify.

cheers,
Rob

Reply:
We already supported the scenario where the Controller had already started `PROVISIONING` a resource.

This change makes it work similarly for when a Controller has already started `DELETING` a resource.

It was found debugging the work for `status_message`.

Reply:
@manstis OK. Thank you for being proactive on the fixes. 

In general, I think it would help reviewers in the future if fixes not related specifically to the feature in development in the PR are done in smaller PRs. It just helps to reduce the amount we have to consider when reviewing.

cheers,
Rob